src/backend/distributed/sql/citus--12.1-1--12.2-1.sql

```diff
@@ -1,5 +1,16 @@
 -- citus--12.1-1--12.2-1
 
+--
+-- citus_internal_database_command creates a database according to the given command.
+
+CREATE OR REPLACE FUNCTION pg_catalog.citus_internal_database_command(command text)
+ RETURNS void
+ LANGUAGE C
+ STRICT
+AS 'MODULE_PATHNAME', $$citus_internal_database_command$$;
+COMMENT ON FUNCTION pg_catalog.citus_internal_database_command(text) IS
+ 'run a database command without transaction block restrictions';
```

Comment:
This should have its own file and that file needs to be included in the upgrade script.

As example, you might want to look into how we added `citus_internal_add_tenant_schema` in 12.0-1 sql version.

Reply:
Thanks. Fixed